src/components/Header/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import bookOpenIcon from './BookOpen.svg';
+import bookmarkIcon from './BOOKMARK_SIMPLE.svg';
```

Comment:
Please make naming for icons consistent.

Reply:
Done